src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml

```diff
@@ -33,7 +33,24 @@ limitations under the License.
     <item name="802.11n" value="0x03"/>
     <item name="802.11ac" value="0x04"/>
     <item name="802.11ax" value="0x05"/>    
+  </enum>
+  <enum name="AssociationFailureCauseType" type="ENUM8">
```

Comment:
Why the "Type" suffix here?  Please just use the spec-defined names?  Similar for the other enums being added here.

Reply:
This is existing style, I removed "Type" suffix for new added here and  https://github.com/project-chip/connectedhomeip/issues/11643 to clean the existing one.

For <enum name="ConnectionStatusType" type="ENUM8">, spec define the same enum for Thread and WiFi, if we use the same name, there will be conflict, is there an common file we can move this definition to?

Reply:
Makes sense to add "Wifi" prefix to disambiguate, as I see you did.